packages/rum-core/src/domain/rumEventsCollection/view/trackScrollMetrics.ts

```diff
@@ -0,0 +1,40 @@
+import type { Duration } from '@datadog/browser-core'
+import { throttle, addEventListener, DOM_EVENT } from '@datadog/browser-core'
+
+export interface ScrollMetrics {
+  maxScrollDepth?: number
+  maxscrollHeight?: number
+  maxScrollDepthTimestamp?: Duration
+}
+
+export function trackScrollMetrics() {
+  const scrollMetrics: ScrollMetrics = {}
+
+  const handleScrollEvent = throttle(
+    (event: Event) => {
+      const scrollTop =
+        window.pageYOffset || (document.documentElement || document.body.parentNode || document.body).scrollTop
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** Use the `getScrollY` function from `packages/rum/src/domain/record/viewports.ts`. Since it is in a different package you'll need to move it (and `getScrollX` to keep them collocated) in `rum-core`, maybe in `rum-core/src/browser/scroll.ts`